src/libraries/System.Private.Xml/src/System/Xml/Xsl/XmlIlGenerator.cs

```diff
@@ -70,6 +70,10 @@ public XmlILGenerator()
         // SxS Note: The way the trace file names are created (hardcoded) is NOT SxS safe. However the files are
         // created only for internal tracing purposes. In addition XmlILTrace class is not compiled into retail
         // builds. As a result it is fine to suppress the FxCop SxS warning.
+        [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2026:MissingRequiresUnreferencedCode",
```

Comment:
(nit) typically I've been using just `IL2026:RequiresUnreferencedCode`. The word after the number doesn't really matter to the trimmer. But warning IL2026 is because you are calling a method attributed with `RequiresUnreferencedCode`. So when you suppress the warning, you are suppressing the fact that you are calling a method with RequiresUnreferencedCode, not that RequiresUnreferencedCode is "missing".